doc/_static/css/nrfxlib.css

```diff
@@ -1,3 +1,76 @@
 :root {
   --docset-color: #ff9933;
 }
+
+/* Doc link box on the ZBOSS page */
+
+.rst-content blockquote.doc-link {
+ padding: 0 10px;
+ display: block;
+ width: 450px;
+ overflow:hidden;
+ transition: all 0.35s ease;
+ box-sizing: border-box;
+ line-height: 50px;
+ vertical-align:middle;
+ position:relative;
+ margin-left: 0;
+}
+
+.doc-link a {
+ color: #fff !important;
+ text-shadow: 0 1px 1px transparent;
+ border-bottom: none;
+ font-size:20px;
+ position:relative;
+ z-index:100;
+}
+
+.doc-link p.doc-link-text {
+ margin-bottom: 0;
+ line-height: inherit;
+}
+
+.doc-link .doc-link-image, .doc-link p.doc-link-text a {
+ float:left;
+ padding: 10px;
+}
+
+.doc-link::after {
+ content: '';
+ position: absolute;
+ background: linear-gradient(29.5deg, transparent 50%, rgba(255, 255, 255, 0.2) 50%);
+ width: 150%;
+ height: 150%;
+ right: 0;
+ top: 30%;
+ left: 70%;
+ transform: translate(-50%, -50%);
+ transition: all 0.35s ease;
+}
+
+.doc-link:hover:after {
+ content: '';
+ background: linear-gradient(29.5deg, transparent 50%, rgba(255, 255, 255, 0.2) 50%);
+ width: 200%;
+ height: 200%;
+ right: 0;
+ top: 50%;
+ left: 50%;
+ transform: translate(-50%, -50%);
+}
+
+.doc-link:active:after {
+ content: '';
+ background: linear-gradient(29.5deg, transparent 50%, rgba(255, 255, 255, 0.2) 50%);
+ width: 200%;
+ height: 200%;
+ right: 0;
+ top: 40%;
+ left: 20%;
+ transform: translate(-50%, -50%);
+}
+
+.yellow-box {
+ background: #ffaa54;
```

Comment:
not sure if it's the color you want exactly, but consider using `var(--docset-color)`

Reply:
I'm using an image in https://github.com/nrfconnect/sdk-nrfxlib/pull/581 that has almost the same color as the docset, so wanted to go for something brighter.

Reply:
OK, added a different "zboss-box" and changed this one to the default docset color, to match "blue-box" in nrf.css.